examples/ResourceGroup/ResourceGroup.Tests.ps1

```diff
@@ -30,3 +30,42 @@ Describe 'Verify Resource Group Does Not Exist' {
     $result.Success | Should -Be $false
   }
 }
+
+Describe 'Verify Resource Group Exists with Custom Assertion' {
+  it 'Should contain an Resource Group named rg-test' {
+    #arrange
+    $rgName = "rg-test"
+
+    #act
+    $result = Confirm-AzBPResourceGroup -ResourceGroupName $rgName
+
+    #assert
+    $result | Should -BeDeployed
+  }
+}
+
+Describe 'Verify Resource Group Exists in Correct Location' {
+  it 'Should contain an Resource Group named rg-test in westus3' {
+    #arrange
+    $rgName = "rg-test"
+
+    #act
+    $result = Confirm-AzBPResourceGroup -ResourceGroupName $rgName
+
+    #assert
+    $result | Should -BeInLocation 'westus3'
+  }
+}
+
+Describe 'Verify Resource Group Exists in Resource Group' {
```

Comment:
Is this a valid test?

Reply:
I kept it in because it worked and if nothing else, proves that that property is in the response object. Down to keep or remove.

Reply:
I'd suggest removing it, only because it doesn't add any new value, and is kind of confusing. Even though it can be done, the `-BeDeployed` test is more appropriate for the intent.